app.js

```diff
@@ -34,26 +31,42 @@ function lockCards(cardsToLock) {
 		card.removeEventListener('click', handleUserInput);
 	});
 	firstCard = null;
+	pausePlay = false;
+
+
 	console.log('someone called lockCards,Cards match');
+
 };
 
 function handleUserInput() {
-	this.classList.add('turned'); // you always want to do this.
+	if (pausePlay) return;
+	this.classList.add('turned');
+	if (this === firstCard) return
 	if (!firstCard) {
-		firstCard = this; //if the firstCard didn't exist, set the first card
+		firstCard = this;
 	} else {
-		currentCard = this; //now it's time to compare both cards!
+		currentCard = this;
```

Comment:
please note that currentCard has not been previously defined, which means this is now a GLOBAL variable (not what you want). So make sure you add a `const` in front of it as to declare it inside this function alone.

Reply:
Oh yes. Silly mistake to make